src/scripts/main.js

```diff
@@ -1,3 +1,261 @@
 'use strict';
 
-// write your code here
+const tbody = document.querySelector('tbody');
+const scoreField = document.querySelector('.game-score');
+
+const btnReset = document.createElement('button');
```

Comment:
Consider using more descriptive variable names instead of `btnReset`. For example, `restartButton`. 🤔